spiceaidocs/docs/reference/datatypes.md

```diff
@@ -0,0 +1,58 @@
+| int8                     | Signed 8-bit little-endian integer.                                          | `TINYINT`                     | `TINYINT`         | `SMALLINT`         |
+| uint16                   | Unsigned 16-bit little-endian integer.                                       | `SMALLINT`                    | `SMALLINT`        | `SMALLINT`         |
+| int16                    | Signed 16-bit little-endian integer.                                         | `SMALLINT`                    | `SMALLINT`        | `SMALLINT`         |
+| uint32                   | Unsigned 32-bit little-endian integer.                                       | `INTEGER`                     | `INT`             | `INTEGER`          |
+| int32                    | Signed 32-bit little-endian integer.                                         | `INTEGER`                     | `INT`             | `INTEGER`          |
+| uint64                   | Unsigned 64-bit little-endian integer.                                       | `BIGINT`                      | `BIGINT`          | `BIGINT`           |
+| int64                    | Signed 64-bit little-endian integer.                                         | `BIGINT`                      | `BIGINT`          | `BIGINT`           |
+| half_float               | 2-byte floating point value                                                  |                               |                   |                    |
+| float                    | 4-byte floating point value                                                  | `FLOAT`                       | `FLOAT`           | `REAL`             |
+| double                   | 8-byte floating point value                                                  | `DOUBLE`                      | `DOUBLE`          | `DOUBLE PRECISION` |
+| string                   | UTF8 variable-length string as List\<Char\>                                  | `VARCHAR`                     | `TEXT`            | `TEXT`             |
+| binary                   | Variable-length bytes (no guarantee of UTF8-ness)                            |                               |                   |                    |
+| fixed_size_binary        | Each value has equal bytes of binary.                                        |                               |                   |                    |
+| date32                   | int32_t days since the UNIX epoch                                            | `DATE`                        | `DATE`            |                    |
+| date64                   | int64_t milliseconds since the UNIX epoch                                    | `DATE`                        | `TIMESTAMP`       |                    |
+| timestamp                | Exact timestamp encoded with int64 since UNIX epoch, seconds or milliseconds | `TIMESTAMP_S`, `TIMESTAMP_MS` | `TIMESTAMP`       |                    |
```

Comment:
I think we support timestamp in postgres? E.g. the taxi trips will be loaded as datetime for a few columns.

Reply:
Good catch, I didn't correctly run the testing for postgres. Will update.